2016-07/july-6.md

```diff
@@ -0,0 +1,56 @@
+## July 6
+
+### Attendees
+
+* [Julen](https://github.com/julen) (Evernote)
+* [Leandro](https://github.com/unho) (Translate House)
+* [Ryan](https://github.com/phlax) (Translate House)
+* [Taras](https://github.com/ta2-1) (Translate House)
+
+
+### Short status and upcoming endeavours report
+
+* 2.8.0b1 was released. The final 2.8.0 will happen before end of July 2016.
+  Bits of `pootle_fs` still need to be landed for that to happen, but it is
+  mostly done.
+* Julen will finish his work on editor using custom font, and then will start
+  working on due dates feature.
+
+
+### Triaging
+
+Some new issues this week.
+
+
+#### Assigned to 2.8
+
+* Popup confirmation always shows up with keyboard submission
```

Comment:
There's no possibility we went through this, as it was filed after the triage meeting.